tests/test_backend.py

```diff
@@ -97,12 +99,17 @@ def tearDown(self):
 
     def test_send_mail(self):
         settings.AWS_SES_CONFIGURATION_SET = None
-        send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
+
+        unicode_from_addr = u'Unicode Name óóóóóó <from@example.com>'
+        # reference: https://tools.ietf.org/html/rfc2047.html
+        rfc2047_encoded_from_addr = '=?utf-8?b?VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==?= <from@example.com>'
```

Comment:
In depth citation, [per python 3 docs](https://docs.python.org/3/library/email.header.html). (Note that the [python 2 counterpart is similar](https://docs.python.org/2/library/email.header.html)):

If you want to include non-ASCII characters in your email headers, say in the Subject or To fields, you should use the Header class and assign the field in the Message object to an instance of Header instead of using a string for the header value. Import the Header class from the email.header module. For example:

```
>>> from email.message import Message
>>> from email.header import Header
>>> msg = Message()
>>> h = Header('p\xf6stal', 'iso-8859-1')
>>> msg['Subject'] = h
>>> msg.as_string()
'Subject: =?iso-8859-1?q?p=F6stal?=\n\n'
```

Notice here how we wanted the Subject field to contain a non-ASCII character? We did this by creating a Header instance and passing in the character set that the byte string was encoded in. When the subsequent Message instance was flattened, the Subject field was properly RFC 2047 encoded. MIME-aware mail readers would show this header using the embedded ISO-8859-1 character.

Reply:
Is it possible to write a function that takes unicode_from_addr and generates the rfc encoded one? Would that be useful to be part of this library to export, or would it be test-only?

Then you could also add a simple unit test for the function. This exact output is fine.

Other than that, this changeset looks good to me!

Reply:
I've added a helper to the tests, it's basically just the relevant portions of [django code](https://github.com/django/django/blob/stable/1.8.x/django/core/mail/message.py#L83-L123) extracted for ease of reading. IMO there's not much value to exporting it as part of this library.